kubernetes/assets/dashboards/kubernetes_clusters.json

```diff
@@ -3790,11 +3790,11 @@
                                     "display_type": "line",
                                     "metadata": [
                                         {
-                                            "alias_name": "network writes per node",
-                                            "expression": "sum:kubernetes.io.write_bytes{$scope,$cluster} by {kube_replica_set,host}-avg:kubernetes_state.replicaset.replicas_ready{$scope,$cluster} by {host}"
+                                            "alias_name": "IO writes per node ",
+                                            "expression": "sum:kubernetes.io.write_bytes{$scope,$cluster} by {kube_replica_set,host}"
```

Comment:
I think we should remove `kube_replica_set` in the `by {}` other widgets like `Network in per node` don't use this tag. And I'm not able to find in the PR that have introduced this dashboard any explanation why it is like this.

Reply:
```suggestion
                                            "expression": "sum:kubernetes.io.write_bytes{$scope,$cluster} by {host}"
```